doc/docs.md

```diff
@@ -251,7 +251,8 @@ In this case `main` doesn't return anything, so there is no return type.
 
 As in many other languages (such as C, Go, and Rust), `main` is the entry point of your program.
 
-[`println`](#println) is one of the few [built-in functions](#builtin-functions).
+[`println`](builtin-functions.html#println) is one of the few 
+[built-in functions](builtin-functions.html).
```

Comment:
This will not work in `docs.md`. There is not a `builtin-functions.html` file in the main repo. That file is only present in the generated output from https://docs.vlang.io/ .